mage_ai/cluster_manager/kubernetes/workload_manager.py

```diff
@@ -144,8 +149,24 @@ def create_workload(
         name: str,
         workspace_config: KubernetesWorkspaceConfig,
         project_type: str = ProjectType.STANDALONE,
-        **kwargs,
```

Comment:
why do you remove the `**kwargs`?

Reply:
i removed the `kwargs` because i'm expecting all the parameters to be passed in through the `workspace_config` argument now

Reply:
Is this code still being used? https://github.com/mage-ai/mage-ai/blob/master/mage_ai/server/api/clusters.py#L129-L134

Reply:
good call i don't think this is used anymore. i'll remove it